src/hooks/useWindowDimensions/index.ts

```diff
@@ -82,7 +91,7 @@ export default function (useCachedViewportHeight = false): WindowDimensions {
         cachedViewportHeightWithKeyboardRef.current = windowHeight;
     }, [isCachedViewportHeight, windowHeight]);
 
-    return {
+    const returnObject = {
```

Comment:
I can see this is later passed as an argument to `lockWindowDimensions` so maybe name it `windowDimensions` or `windowDimensionsObject`?